NitroxModel/Helper/ReflectionHelper.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace NitroxModel.Helper
+{
+    public static class ReflectionHelper
```

Comment:
I would do Validate.NotNull(var, o.getType() + " does not have a x named" + name) instead of allowing these functions to silently fail by returning null.  Makes the code cleaner(no null check) and is a controlled failure(no NRE in the caller).  Also better naming for mi/fi would be good.